services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaServiceITest.kt

```diff
@@ -844,36 +956,29 @@ class HankeKayttajaServiceITest : DatabaseTest() {
     }
 
     private fun Assert<List<KayttajaTunnisteEntity>>.areValid() = each { t ->
-        t.transform { it.id }.isNotNull()
-        t.transform { it.kayttooikeustaso }.isEqualTo(Kayttooikeustaso.KATSELUOIKEUS)
-        t.transform { it.createdAt }.isRecent()
-        t.transform { it.sentAt }.isNull()
-        t.transform { it.tunniste }.matches(Regex(kayttajaTunnistePattern))
-        t.transform { it.hankeKayttaja }.isNotNull()
+        t.prop(KayttajaTunnisteEntity::id).isNotNull()
+        t.prop(KayttajaTunnisteEntity::kayttooikeustaso).isEqualTo(Kayttooikeustaso.KATSELUOIKEUS)
+        t.prop(KayttajaTunnisteEntity::createdAt).isRecent()
+        t.prop(KayttajaTunnisteEntity::sentAt).isNull()
+        t.prop(KayttajaTunnisteEntity::tunniste).matches(Regex(kayttajaTunnistePattern))
+        t.prop(KayttajaTunnisteEntity::hankeKayttaja).isNotNull()
     }
 
     private fun Assert<List<HankeKayttajaEntity>>.areValid(hankeId: Int?) = each { k ->
-        k.transform { it.id }.isNotNull()
-        k.transform { it.nimi }.isIn(*expectedNames)
-        k.transform { it.sahkoposti }.isIn(*expectedEmails)
-        k.transform { it.hankeId }.isEqualTo(hankeId)
-        k.transform { it.permission }.isNull()
-        k.transform { it.kayttajaTunniste }.isNotNull()
+        k.prop(HankeKayttajaEntity::id).isNotNull()
+        k.prop(HankeKayttajaEntity::nimi).isIn(*expectedNames)
+        k.prop(HankeKayttajaEntity::sahkoposti).isIn(*expectedEmails)
+        k.prop(HankeKayttajaEntity::hankeId).isEqualTo(hankeId)
+        k.prop(HankeKayttajaEntity::permission).isNull()
+        k.prop(HankeKayttajaEntity::kayttajaTunniste).isNotNull()
     }
 
     private fun Assert<PermissionEntity>.isOfEqualDataTo(other: PermissionEntity) =
```

Comment:
This seems to be an unused function now.

Reply:
Thank you, removed.